File: engine/model/src/main/scala/tools/packet/character/SetAutoMpPot.scala
package tools.packet.character

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class SetAutoMpPot(private var _itemId: Int) extends PacketInput {
  def itemId: Int = _itemId

  override def opcode(): SendOpcode = SendOpcode.AUTO_MP_POT
}

File: engine/engine-model/src/main/java/tools/packet/character/SetAutoMpPot.java
package tools.packet.character;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record SetAutoMpPot(Integer itemId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.AUTO_MP_POT;
   }
}